extensions/iidm/test/CoordinatedReactiveControlTest.cpp

```diff
@@ -67,16 +67,13 @@ BOOST_AUTO_TEST_CASE(CoordinatedReactiveControlTest) {
     Network network = createNetwork();
     Generator& gen = network.getGenerator("GEN");
 
-    POWSYBL_ASSERT_THROW(CoordinatedReactiveControl(gen, -1.0), PowsyblException, "Unexpected value for qPercent: -1");
-    POWSYBL_ASSERT_THROW(CoordinatedReactiveControl(gen, 101.0), PowsyblException, "Unexpected value for qPercent: 101");
```

Comment:
Why don't you test that setting invalid values do not throw exceptions anymore?

Reply:
You're right, I "restored" them.